ext/cl_khr_defined_builtin_kernels.asciidoc

```diff
@@ -37,10 +37,10 @@ This extension is written against the OpenCL Specification version 3.0.12.
 
 This extension requires OpenCL 1.2 or later.
 
-This extension requires cl_khr_tensor.
+This extension requires cl_exp_tensor.
 
 [NOTE]
-cl_khr_tensor is unpublished, work-in-progress
+cl_exp_tensor is unpublished, work-in-progress
```

Comment:
Not really unpublished, it's being edited publicly and will be pushed to the public Khronos repo as a PR.